docs/Configuration.md

```diff
@@ -298,46 +309,44 @@ We require that all builds associated with a particular application be labeled w
 
 Currently we support GitHub and GitLab, with BitBucket coming soon. Open an issue or a pull request to add support for additional Git providers!
 
-
-#### Instance Config
+#### Instance config
 
 ```yaml
 exporters:
   instances:
   - app_name: committime-exporter
     exporter_type: committime
     env_from_secrets:
-    - github-secret
+    - GitHub-secret
     env_from_configmaps:
     - pelorus-config
     - committime-config
 ```
 
 #### ConfigMap Data Values
 
-This exporter provides several configuration options, passed via `pelorus-config` and `committime-config` variables. User may define own ConfigMaps and pass to the committime exporter in a similar way.
+This exporter provides several configuration options that are passed using `pelorus-config` and `committime-config` variables. Users can define their own ConfigMaps and pass them to the committime exporter in a similar way.
 
 | Variable | Required | Explanation | Default Value |
 |---|---|---|---|
-| `API_USER` | yes | User's github username | unset |
-| `TOKEN` | yes | User's Github API Token | unset |
-| `GIT_API` | no | GitHub, Gitea or Azure DevOps API FQDN. This allows the override for Enterprise users. Currently only applicable to `github`, `gitea` and `azure-devops` provider types. | `api.github.com`, or `https://try.gitea.io`. No default for Azure DevOps. |
-| `GIT_PROVIDER` | no | Set Git provider type. Can be `github`, `gitlab`, or `bitbucket` | `github` |
-| `LOG_LEVEL` | no | Set the log level. One of `DEBUG`, `INFO`, `WARNING`, `ERROR` | `INFO` |
-| `APP_LABEL` | no | Changes the label key used to identify applications  | `app.kubernetes.io/name`  |
-| `NAMESPACES` | no | Restricts the set of namespaces from which metrics will be collected. ex: `myapp-ns-dev,otherapp-ci` | unset; scans all namespaces |
-| `PELORUS_DEFAULT_KEYWORD` | no | ConfigMap default keyword. If specified it's used in other data values to indicate "Default Value" should be used | `default` |
-| `COMMIT_HASH_ANNOTATION` | no | Annotation name associated with the Build from which hash is used to calculate commit time | `io.openshift.build.commit.id` |
-| `COMMIT_REPO_URL_ANNOTATION` | no | Annotation name associated with the Build from which GIT repository URL is used to calculate commit time | `io.openshift.build.source-location` |
-  
+| `API_USER` | yes | User's GitHub username | unset |
+| `TOKEN` | yes | User's GitHub API Token | unset |
+| `GIT_API` | no | GitHub, Gitea or Azure DevOps API FQDN. Allows override for Enterprise users. Currently only applicable to `GitHub`, `gitea` and `azure-devops` provider types. | `api.GitHub.com`, or `https://try.gitea.io`. No default for Azure DevOps. |
+| `GIT_PROVIDER` | no | Set Git provider type. Can be `GitHub`, `gitlab`, or `bitbucket` | `GitHub` |
```

Comment:
same here github not GitHub, we could rename it to GitHub GitLab Bitbucket, but then code needs to be updated as well